test/test_oneview_logical_enclosure_script_facts.py

```diff
@@ -57,9 +57,8 @@ def test_should_get_an_logical_enclosure_script(self, mock_ansible_module,
 
     @mock.patch.object(OneViewClient, 'from_json_file')
     @mock.patch('oneview_logical_enclosure_script_facts.AnsibleModule')
-    def test_should_get_an_logical_enclosure_script_raises_error(self,
-                                                                 mock_ansible_module,
-                                                                 mock_ov_client_from_json_file):
+    def test_should_fail_when_get_logical_enclosure_script_raises_exception(self, mock_ansible_module,
+                                                                            mock_ov_client_from_json_file):
         mock_ov_instance = mock.Mock()
```

Comment:
The method that raised the exception was the 'get_by', not the 'get_logical_enclosure_script'